### ISSUE TITLE
Change handling of non existing networks and containers for accept rules

### ISSUE THREAD

**Issue by thielepaul** [pitkley/dfw#166]
Let's assume a configuration like the following:
```
[container_to_wider_world]
default_policy = "drop"

[[container_to_wider_world.rules]]
network = "net"
src_container = "container"
verdict = "accept"
```

If neither the network `net` nor the container `container` exist, the resulting rule on my ps is:
`oifname "wlp58s0" meta mark set 0x000000df accept`
This rule allows traffic from any docker container regardless of the default policy.

If the network does exist but the container does not the result is:
`iifname "br-bfb96f179176" oifname "wlp58s0" meta mark set 0x000000df accept`
This only allows traffic from containers in the specified network. Better, but still more permissive than intended.

Only, if the a container with the matching name exists in the correct network, the rule looks like expected:
`ip saddr 172.21.0.2 iifname "br-bfb96f179176" oifname "wlp58s0" meta mark set 0x000000df accept`

My suggestion is:
* to either fail for the rule if the network or container it matches is not available such that the user notices that something is not as expected
* or to use sensible defaults: e.g. do not apply accept rules if these cannot be matched to containers/networks, but do apply drop/reject rules the same way it is done right now.

I prefer the first approach as it makes sure the user notices the problem in the setup

**Comment by pitkley** [comment]
@thielepaul thank you very much for reporting this, this is indeed not expected behaviour. Turns out that some validation-functionality was lost in the transition to nftables and I have now included it again (addd063). Additionally the logic for adding a rule if the network exists but the container doesn't was simply flawed so I fixed that, too (e4fb869).

For the cases you have described, the behaviour should now be as follows:

* "If neither the network `net` nor the container `container` exist" DFW will now fail with an error again.
* "If the network does exist but the container does not" DFW will now fail with an error.
* If both the network and the container exist, DFW will add the rule as expected.

The Docker-Hub image build is queued, knowing previous times it will probably take 1-2 hours until the image is updated.

Can you confirm that this fixes the issue? :+1:

**Comment by thielepaul** [comment]
Nice work! It does fix the issue for me :+1:
